src/main/java/org/acra/config/ConfigurationBuilder.java

```diff
@@ -692,8 +692,9 @@ public ConfigurationBuilder setKeyStoreFactory(KeyStoreFactory keyStoreFactory)
         return this;
     }
 
+    @SafeVarargs
     @NonNull
-    public ConfigurationBuilder setReportSenderFactoryClasses(@NonNull Class<? extends ReportSenderFactory>[] reportSenderFactoryClasses) {
+    public final ConfigurationBuilder setReportSenderFactoryClasses(@NonNull Class<? extends ReportSenderFactory>... reportSenderFactoryClasses) {
```

Comment:
http://stackoverflow.com/questions/27208570/why-cant-safevarags-be-applied-to-instance-methods-in-a-final-class